[word] (Requirement Sets) Splitting Word requirement set pages (#1185)
OfficeDev/office-js-docs-pr: docs/reference/requirement-sets/word-preview-apis.md

---
title: Word JavaScript preview APIs
description: 'Details about upcoming Word JavaScript APIs'
ms.date: 07/17/2019
ms.prod: word
localization_priority: Normal
---

# Word JavaScript preview APIs

New Word JavaScript APIs are first introduced in "preview" and later become part of a specific, numbered requirement set after sufficient testing occurs and user feedback is acquired.

> [!NOTE]
> Preview APIs are subject to change and are not intended for use in a production environment. We recommend that you try them out in test and development environments only. Do not use preview APIs in a production environment or within business-critical documents.
>
> To use preview APIs, you must reference the **beta** library on the CDN (https://appsforoffice.microsoft.com/lib/beta/hosted/office.js) and you may also need to join the Office Insider program to get a recent Office build.

> ## API List

Reviewer: ```suggestion
## API list
```

> The following is a complete list of APIs currently in preview.

Reviewer: ```suggestion
The following table lists the APIs that are currently in preview.
```
